systemtest/src/main/java/io/strimzi/systemtest/kafkaclients/IKafkaClientOperations.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.kafkaclients;
+
+public interface IKafkaClientOperations<T> {
```

Comment:
Why are we naming interfaces with `I` prefix here, when it's not something we do in the rest of the code base? Can't you think of a better name without the `I`?